src/index.js

```diff
@@ -0,0 +1,199 @@
+const Debug = require('debug')
+const AWS = require('aws-sdk')
+const { sleep } = require('./util')
+
+const debug = Debug('dynamodb:raziel')
```

Comment:
can we make the debug string just `raziel`? We already take that into account in our serverless repo.